### DISCUSSION TITLE
Winget Availability

### DISCUSSION THREAD

**Discussion by saigkill** [BurntSushi/ripgrep#2185] [General] [answered]
Hello. I just would like to ask, if ripgrep will be available with winget?

**Comment by BurntSushi** [comment, marked as answer]
I don't know, sorry. I develop ripgrep and make binary releases available on GitHub. Others do the work of making ripgrep available in package managers, and they will typically send pull requests updating the README with install instructions.

Maybe a Windows user can help answer your question.

See also: https://github.com/BurntSushi/ripgrep/issues/1637
